07-reputation/03-localtrust.md

```diff
@@ -0,0 +1,65 @@
+## Local Trust
+
+*Local trust*
+: trust of one \Gls{Node} to another, calculated using *only* statistical information of their peer-to-peer network interactions. The Subject and object of such a trust are peer-to-peer nodes.
+
+### Subject and Object of a trust
+
+Any node is the Subject of evaluation. The object of trust is any node other than the Subject. In \Gls{NeoFS}, nodes do not evaluate their own performance, because the default is “every node trusts itself”.
+
+The evaluation criterion is the successful (non error in sense of network interaction) execution of one of the RPC calls:
+
+1. `Put`
+2. `Get`
+3. `Search`
+4. `GetRange`
+5. `GetRangeHash`
+6. `Head`
+7. `Delete`
+
+A Subject of the Local trust assessment stores an assessment of the experience of communication with the Objects in `LocalStorage` while interacting with them. Local reputation only makes sense in context with some kind of epoch value, which should be taken into account when storing and transferring values.
+
+### Calculating trust
+
+All formulations, terms and statements in this clause take into account that Local trust makes sense only in the context of a certain epoch, that is, peer-to-peer interactions of nodes for an epoch `n` affect *only* local trusts calculated in the epoch `n`. That fact is not emphasized explicitly further.
```

Comment:
Using all these formulations, terms and statements, we recognize that Local Trust is only applicable in the context of a certain epoch. It means that peer-to-peer interactions of nodes for an epoch `n` affect *only* Local Trusts calculated in the epoch `n`.

Reply:
fixed